src/sorunlib/stimulator.py

```diff
@@ -0,0 +1,126 @@
+import time
+import sorunlib as run
+
+ID_SHUTTER = 1
+
+
+def _open_shutter():
+    """Open the shutter of the stimulator"""
+    ds = run.CLIENTS['stimulator']['ds378']
+    ds.set_relay(relay_number=ID_SHUTTER, on_off=1)
+    time.sleep(1)
+
+
+def _close_shutter():
+    """Close the shutter of the stimulator"""
+    ds = run.CLIENTS['stimulator']['ds378']
+    ds.set_relay(relay_number=ID_SHUTTER, on_off=0)
+    time.sleep(1)
+
+
+def _setup():
+    # Open shutter
+    _open_shutter()
+    time.sleep(1)
+
+    # Acceleration / Decceleration configuration
+    blh = run.CLIENTS['stimulator']['blh']
+    blh.set_value(accl_time=10)
+    blh.set_value(decl_time=10)
+
+
+def _stop():
+    blh = run.CLIENTS['stimulator']['blh']
+
+    # Stop rotation
+    blh.stop_rotation()
+    time.sleep(10)
+
+    # Close shutter
+    _close_shutter()
+
+
+def calibrate_tau(duration_step=10,
+                  speeds_rpm=[225, 495, 945, 1395, 1845, 2205],
+                  forward=True, do_setup=True, stop=True):
+    """Time constant calibration using the stimulator.
+
+    Parameters
+    ----------
+    duration_step : float, optional
+        Duration of each step of time constant measurement in sec, default to 10 sec.
+    speeds_rpm : list of float, default [225, 495, 945, 1395, 1845, 2205].
+        List of chopper rotation speed in RPM for each step.
+    forward : bool, default True
+        Chopper rotation direction. True for clockwise rotation.
```

Comment:
It'd be great to specify what orientation clockwise is considered from, i.e. out from the stimulator or looking at the stimulator from the receiver.

Reply:
Thanks for adding this. I added this same description to `calibrate_tau`, which also has the 'forward' argument.